Source/OktaAuthSdk.swift

```diff
@@ -17,10 +17,11 @@ public class OktaAuthSdk {
     public class func authenticate(with url: URL,
                                    username: String,
                                    password: String?,
+                                   httpClient: OktaAuthHTTPClient? = nil,
```

Comment:
Please remove it here. 99% of devs will never use OktaAuthHTTPClient, don't confuse them with this parameter